src/terminal.go

```diff
@@ -1105,7 +1105,12 @@ func keyMatch(key int, event tui.Event) bool {
 
 func quoteEntry(entry string) string {
 	if util.IsWindows() {
-		return strconv.Quote(strings.Replace(entry, "\"", "\\\"", -1))
+		escaped := strings.Replace(entry, "\\", "\\\\", -1)
+		escaped = "\"" + strings.Replace(escaped, "\"", "\\\"", -1) + "\""
+		r, _ := regexp.Compile("[&|<>()@^%\"]")
```

Comment:
Let's use raw string literals (using backticks) in these lines to make them more readable.

https://golang.org/ref/spec#String_literals